src/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Phantoms</title>
+</head>
+<body>
+    <div id="root"></div>
+</body>
+</html>
```

Comment:
why do you have this here>?

Reply:
it was a slight problem with rebasing, but I just solved it!.